qa/rpc-tests/cfund-proposal-reorg.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# Copyright (c) 2019 The Navcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import NavCoinTestFramework
+from test_framework.cfund_util import *
+
+import urllib.parse
+
+class CommunityFundProposalReorg(NavCoinTestFramework):
+    def __init__(self):
+        super().__init__()
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+
+    def setup_network(self, split=False):
+        self.nodes = []
+
+        self.nodes.append(start_node(0, self.options.tmpdir, []))
+        self.nodes.append(start_node(1, self.options.tmpdir, []))
+
+        connect_nodes_bi(self.nodes, 0, 1)
+
+        self.is_network_split = split
+
+    def run_test(self):
+
+        activate_cfund(self.nodes[0])
+        sync_blocks(self.nodes)
+
+        rawproposal = self.nodes[0].createproposal(self.nodes[0].getnewaddress(), 10, 36000, "test", 50, True)
+
+        # disconnect the nodes and generate the payout on each node
+        url = urllib.parse.urlparse(self.nodes[1].url)
+        self.nodes[0].disconnectnode(url.hostname+":"+str(p2p_port(1)))
+
+        hash = self.nodes[0].sendrawtransaction(rawproposal)
+        self.nodes[1].sendrawtransaction(rawproposal)
+
+        self.nodes[0].generate(1)
+        self.nodes[1].generate(2)
+
+        connect_nodes_bi(self.nodes, 0, 1)
+        sync_blocks(self.nodes)
+
+        assert(self.nodes[0].getproposal(hash) == self.nodes[0].getproposal(hash))
```

Comment:
i think this is supposed to compare node 0 to node 1, not just node 0 to node 0 ?

I'm trying to add checking the payment request reorg to this test, so i can fix it in my commit.